CONTRIBUTING.md

```diff
@@ -75,29 +80,78 @@ Note that the FLS ID should be filled according to the FLS paragraph ID for whic
 
 You would then pull `fls_4rhjpdu4zfqj` to place in the FLS ID field.
 
-### 2) Create a Draft with a Member
 
-A member of the Coding Guidelines Subcommittee should get you a first review with some feedback within 14 days of submission. You'll work with one or more members to flesh out the concept and ensure the guideline is well prepared.
+### 2) Guideline Generated as a Comment
+
+A GitHub Action will fire, adding a comment to your newly created issue with
+the contents of the coding guideline prepared written out correctly
+in reStructured Text.
```

Comment:
We should mention that this comment gets updated with every edit. It's a really useful feature you guys pulled off which makes contributing so much easier.

Reply:
(PS: I would suggest an edit myself, but good god I'm so tired right now)